apps/snmp_exporter/app.yaml

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "/snmp_exporter",
+    "cmd": null,
+    "cpus": 1,
+    "mem": 1024,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
+    "executor": "",
+    "requirePorts": false,
+    "constraints": [
+        [
+            "secrets",
+            "LIKE",
+            "true"
+        ],
+    ],
+    "acceptedResourceRoles": ['*'],
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "docker.ocf.berkeley.edu/snmp_exporter",
+            "privileged": false,
+            "parameters": [],
+            "forcePullImage": false
+        },
+        "portMappings": [
+            {
+                "containerPort": 9116,
+                "hostPort": 0,
+                "servicePort": 10013,
+                "protocol": "tcp",
+                "name": "main",
+                "labels": {}
+            }
+        ]
+    },
+    "networks": [
+        {
+            "mode": "container/bridge"
+        }
+    ],
+    "healthChecks": [
+        {
+            "path": "/",
+            "protocol": "MESOS_HTTP",
+            "portIndex": 0,
+            "delaySeconds": 300,
+            "gracePeriodSeconds": 300,
+            "intervalSeconds": 60,
+            "timeoutSeconds": 20,
+            "maxConsecutiveFailures": 3,
+            "ipProtocol": "IPv4",
+        }
+    ],
+    "labels": {
+        "HAPROXY_GROUP": "lb"
```

Comment:
I don't think this is necessary either

Reply:
I think this is probably still necessary, so that the load balancer picks up this service and knows to serve it at whatever port it is assigned (the service port). Otherwise, I don't believe that's done.